### DISCUSSION TITLE
Create temporary request

### DISCUSSION THREAD

**Discussion by liketoeatcheese** [usebruno/bruno#3262] [Ideas]
Hi team,

Sometimes, people just need a quick and dirty request to test something without the need to save it. But Bruno enforced to create a file for this. Is there a way to do a temporary request without having to create it in a collection?

I still need to open Postman to do those quick ones which I'm trying to avoid :)

**Comment by drchiodo** [comment]
i solved in this way
![image](https://github.com/user-attachments/assets/2898568d-4cf6-41fb-91c9-6158b803c9b1)

**Reply by liketoeatcheese** [reply]
Yea that's what I do sometimes as well. But going though the process of creating a request with naming and everything is just unnecessary. A better process in my opinion is that when you create a request with a plus button, it will just be a untitled-xxx request that's automatically created in /tmp folder like `/tmp/bruno_untitled_xxx`. And if the user save, it will then prompt user to specify the name as well as the location of the request. Else there will be a cron job running every day to just clean up any bruno tmp file that's not being active than a day. Using this strat, temp file can still be recovered and save when need be

**Comment by Asuza** [comment]
I would love this feature! I want to be able to open a single-use request from another system to edit in Bruno and then send. As far as I know, there's no way for Bruno to do that.
